core/src/main/scala/org/scalarelational/column/ColumnLike.scala

```diff
@@ -18,7 +18,7 @@ trait ColumnLike[T] extends SelectExpression[T] with ColumnPropertyContainer {
   def dataType: DataType[T]
   def manifest: Manifest[T]
 
-  def sqlType = dataType.sqlType(table.datastore, this)
+  def sqlType = dataType.dbType
```

Comment:
I think I should rename that method to `dbType`. As I said, I can't use `SQLType` because of namespace collision.